Makefile.core.mk

```diff
@@ -625,7 +625,7 @@ bundle: gen-all-except-bundle helm operator-sdk ## Generate bundle manifests and
 	if [ "$(OPENSHIFT_PLATFORM)" = "true" ]; then \
 		TEMPL_FLAGS="$$TEMPL_FLAGS --set platform=openshift"; \
 	fi; \
-	$(HELM) template chart chart $$TEMPL_FLAGS --set image='$(IMAGE)' --set bundleGeneration=true | $(OPERATOR_SDK) generate bundle $(BUNDLE_GEN_FLAGS)
+	$(HELM) template chart chart $$TEMPL_FLAGS --set bundleGeneration=true | $(OPERATOR_SDK) generate bundle $(BUNDLE_GEN_FLAGS)
```

Comment:
We will have to do this upstream to avoid conflicts when syncing upstream changes. I will prepare a PR